redash/query_runner/__init__.py

```diff
@@ -70,6 +73,34 @@ def type(cls):
     def enabled(cls):
         return True
 
+    @property
+    def host(self):
+        if "host" in self.configuration:
+            return self.configuration["host"]
+        else:
+            raise NotImplementedError()
+
+    @host.setter
+    def host(self, host):
+        if "host" in self.configuration:
+            self.configuration["host"] = host
+        else:
+            raise NotImplementedError()
+
+    @property
+    def port(self):
+        if "port" in self.configuration:
+            return self.configuration["port"]
+        else:
+            raise NotImplementedError()
+
+    @port.setter
+    def port(self, port):
+        if "port" in self.configuration:
+            self.configuration["port"] = port
+        else:
+            raise NotImplementedError()
```

Comment:
Let's add a comment explaining the purpose of these properties.

Reply:
👍 8da215b3